<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-G790OGBU" name="com.menumitratCommonAPITestScript.UpdateOrderTestScript" tests="7" failures="0" timestamp="9 May 2025 06:01:24 GMT" time="2.265" errors="2">
  <testcase name="updateOrderNegativeTest" time="0.127" classname="com.menumitratCommonAPITestScript.UpdateOrderTestScript"/>
  <testcase name="updateDriveThroughOrder" time="0.345" classname="com.menumitratCommonAPITestScript.UpdateOrderTestScript"/>
  <testcase name="updateCounterOrder" time="0.357" classname="com.menumitratCommonAPITestScript.UpdateOrderTestScript"/>
  <testcase name="updateParcelOrder" time="0.380" classname="com.menumitratCommonAPITestScript.UpdateOrderTestScript"/>
  <testcase name="updateOrderNegativeTest" time="0.125" classname="com.menumitratCommonAPITestScript.UpdateOrderTestScript">
    <error type="com.menumitra.utilityclass.customException" message="Error in update order negative test: A JSONObject text must begin with &#039;{&#039; at 1 [character 2 line 1]">
      <![CDATA[com.menumitra.utilityclass.customException: Error in update order negative test: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at com.menumitratCommonAPITestScript.UpdateOrderTestScript.updateOrderNegativeTest(UpdateOrderTestScript.java:945)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- updateOrderNegativeTest -->
  <testcase name="updateDeliveryOrder" time="0.376" classname="com.menumitratCommonAPITestScript.UpdateOrderTestScript"/>
  <testcase name="updateDineInOrder" time="0.555" classname="com.menumitratCommonAPITestScript.UpdateOrderTestScript">
    <error type="com.menumitra.utilityclass.customException" message="Exception in update Dine-In order test: Status code mismatch - Expected: 200, Actual: 400">
      <![CDATA[com.menumitra.utilityclass.customException: Exception in update Dine-In order test: Status code mismatch - Expected: 200, Actual: 400
	at com.menumitratCommonAPITestScript.UpdateOrderTestScript.updateDineInOrder(UpdateOrderTestScript.java:368)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- updateDineInOrder -->
</testsuite> <!-- com.menumitratCommonAPITestScript.UpdateOrderTestScript -->
